.github/workflows/int-and-connected-test-run.yml

```diff
@@ -125,3 +125,14 @@ jobs:
         uses: broadinstitute/datarepo-actions@0.35.0
         with:
           actions_subcommand: 'gcp_whitelist_clean'
+      - name: "Notify Jade Slack"
```

Comment:
I thinking a slightly more detailed name might be appropriate here since we use this workflow in so many places.. something like "Notify jade slack on nightly test run"